augur/server.py

```diff
@@ -16,24 +16,14 @@
 import json
 
 AUGUR_API_VERSION = 'api/unstable'
-# Location to load configuration from
-CONFIG_BAD = False
+
+
 '''
 make a try and accept condition
 if its open the GH_DATA_ONFIG_FILE and then its open in read mode
 and if the file does't open the it print Couldn\'t open config file, attempting to create.
 '''
 
-try:
```

Comment:
@howderek : Maybe I am reading this wrong, but I don't think we want to delete this stuff from server.py ...